### DISCUSSION TITLE
希望能增加腾讯翻译君术语库和例句库的使用

### DISCUSSION THREAD

**Discussion by Kunlineaten** [immersive-translate/immersive-translate#1903] [Ideas]
### 新功能 | New Feature

官方介绍文档：https://cloud.tencent.com/document/product/551/15619
1.术语库参数：“TermRepoIDList.N”
2.例句库参数："SentRepoIDList.N"

### 为什么您想要添加这些功能 | Why you want to add these features

术语库官方文档：https://cloud.tencent.com/document/product/551/107926
例句库官方文档：https://cloud.tencent.com/document/product/551/107927

**Comment by theowenyoung** [comment]
我们研究一下～
